CardForm/src/test/resources/robolectric.properties

```diff
@@ -1,2 +1 @@
-sdk=21
-constants=com.braintreepayments.cardform.BuildConfig
\ No newline at end of file
+sdk=28
```

Comment:
we're updating here to 28 and not 30, is there a reason why?

Reply:
Yes Robolectric doesn't support API 30 yet.  We are still bumping this to 28 from 21 to be in line with the other repositories.

Reply:
For Robolectric Android 29, we would have to switch to Java 9. The downside here is that there is no explicit support for Java 9 in Android Studio. And Android 30 support for Robolectric is still in [alpha](https://github.com/robolectric/robolectric/releases) 😞  .